app/lib/model/api_server/GraphQlServerAccess.dart

```diff
@@ -482,36 +484,48 @@ Additive? _convertAdditive(Enum$Additive e) {
     Enum$Additive.LAXATIVE_IF_OVERUSED => Additive.laxativeIfOverused,
     Enum$Additive.PHENYLALANINE => Additive.phenylalanine,
     Enum$Additive.ALCOHOL => Additive.alcohol,
-    Enum$Additive.PRESSED_MEET => Additive.pressedMeat,
+    Enum$Additive.PRESSED_MEAT => Additive.pressedMeat,
```

Comment:
`nutrition/logic` seems to have diverged from `main` and didn't compile, so I manually applied a couple of changes like this to get a working app to start with.